### DISCUSSION TITLE
Is M808 loop command supported?

### DISCUSSION THREAD

**Discussion by TRX0011** [mriscoc/Ender3V2S1#856] [Q&A] [answered]
Thinking to turn my Ender 3v2 into a conveyor belt, and Marlin has a loop command, the M808 L. / M808
Does Professional firmware support it? 

Anyone tried this command? Is Loop count work as supposed to?

**Comment by mriscoc** [comment, marked as answer]
The precompiled versions don't have GCODE_REPEAT_MARKERS enabled, so you can't use M808. You will need to try a custom build.

**Reply by TRX0011** [reply]
Thank you for letting me know.
